lib/marqeta/api_object.rb

```diff
@@ -0,0 +1,40 @@
+module Marqeta
+  class ApiObject
```

Comment:
Would `ApiResponse` (or even just `Response`) be clearer here?

Reply:
Ok, now that I scroll down, I see it's has aspects of the the request and response...